ext/ManifoldsTestExt/tests_general.jl

```diff
@@ -603,7 +603,7 @@ function test_manifold(
                 end
                 # check projection idempotency
                 for i in 1:N
-                    Test.@test isapprox(M, p, project(M, p, bvectors[i]), bvectors[i])
+                    Test.@test isapprox(M, p, project(M, p, bvectors[i]), p \ bvectors[i])
```

Comment:
This is a generic test, so you can't modify it like that. It's also used for manifolds that don't do this translation to identity. Easiest fix would be disabling or special-casing this test for GL(N).

Reply:
Oh, yes, I was too fast there. My usual problem – thinking I can fix things fast. Then this has to wait until I find time again.

Reply:
I at least did an undo on this. But then someone (tm) still has to adapt the tests to make this pass again. I added a todo in the PR.

Reply:
That is a bit hacked, but given that this is mainly on deprecated code probably fine.

Reply:
It will be removed in the next breaking version anyway, so I don't care that much.